Build failing:
```
FAIL: corehq.apps.userreports.tests.test_pillow:RebuildTableTest.test_ordered_pk
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_pillow.py", line 849, in test_ordered_pk
    config.save()
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/models.py", line 197, in save
    super(DataSourceConfiguration, self).save(**params)
  File "/mnt/commcare-hq-ro/corehq/apps/cachehq/mixins.py", line 20, in save
    self.clear_caches()
  File "/mnt/commcare-hq-ro/corehq/apps/cachehq/mixins.py", line 62, in clear_caches
    super(QuickCachedDocumentMixin, self).clear_caches()
  File "/mnt/commcare-hq-ro/corehq/apps/cachehq/mixins.py", line 14, in clear_caches
    invalidate_document(self)
  File "/mnt/commcare-hq-ro/corehq/apps/cachehq/invalidate.py", line 16, in invalidate_document
    _get_cache_processor().process_doc(document.to_json(), deleted)
  File "jsonobject/base.pyx", line 259, in jsonobject.jsonobject.base.JsonObjectBase.to_json
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/models.py", line 400, in validate
    columns = [c.id for c in self.indicators.get_columns()]
  File "/vendor/lib/python2.7/site-packages/memoized.py", line 13, in _inner
    return m(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/memoized.py", line 133, in __call__
    value = self.func( *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/models.py", line 336, in indicators
    for indicator in self.configured_indicators
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/indicators/factory.py", line 175, in from_spec
    return cls.constructor_map[spec['type']](spec, context)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/indicators/factory.py", line 52, in _build_raw_indicator
    getter=wrapped.getter,
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/indicators/specs.py", line 82, in getter
    getter = getter_from_property_reference(self)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/expressions/getters.py", line 186, in getter_from_property_reference
    assert spec.property_path, spec.property_name
AssertionError: None
```

--- a/corehq/apps/userreports/tests/test_pillow.py
+++ b/corehq/apps/userreports/tests/test_pillow.py
@@ -844,6 +844,7 @@ class RebuildTableTest(TestCase):
             "column_id": "pk_key",
             "type": "raw",
             "datatype": "string",
+            "property_name": "owner_id",
             "is_primary_key": True
         })
         config.save()

